streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamJoinTest.java

```diff
@@ -211,88 +197,73 @@ public void testOuterJoin() {
         assertEquals(1, copartitionGroups.size());
         assertEquals(new HashSet<>(Arrays.asList(topic1, topic2)), copartitionGroups.iterator().next());
 
-        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props, 0L)) {
-
+        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props)) {
```

Comment:
No need to set initial wall-clock time of the driver (removed to avoid confusion with event-time). Similar on other places.